docs/docs/middlewares/image.md

```diff
@@ -0,0 +1,25 @@
+# Image
+This component returns path to image in `image_cache` directory. It also offers edit functions thanks Gregwar.
```

Comment:
please reword:
It also offers edit functions thanks Gregwar. => The output image can be modified by functions provided by `Gregwar/Image` library.